README.md

```diff
@@ -74,6 +95,8 @@ codePush.sync();
 
 If an update is available, it will be silently downloaded, and installed the next time the app is restarted (either explicitly by the end user or by the OS), which ensures the least invasive experience for your end users. If you would like to display a confirmation dialog (an "active install"), or customize the update experience in any way, refer to the sync method's API reference for information on how to tweak this default behavior.
 
+*NOTE: While [Apple's developer agreement](https://developer.apple.com/programs/ios/information/iOS_Program_Information_4_3_15.pdf) fully allows performing over-the-air updates of JavaScript and assets (which is what enables CodePush!), it is against their policy for an app to display an update prompt. Because of this, we recommend that App Store-distributed apps don't enable the `updateDialog` option when calling `sync`, whereas Google Play and internally distributed apps (e.g. Enterprise, Fabric, HockeyApp) can choose to enable/customize it.*
```

Comment:
Devs can definitely release an update that enables the dialog and/or always use the dialog and just make sure to not release to CodePush until after the app is approved, but I'm not sure if I want to make that recommendation here :)

We know for a fact that the update dialog is against Apple's policies, and we've now seen 5 apps rejected because of it (they were all approved after removing the dialog), so I want to be really conservative in the prescription we give.

Since you already "convert" mandatory updates to immediately install, the Cordova plugin has a pretty solid "silent mode" experience, and I want to mirror that behavior in the RN plugin to ensure that we optimize for apps that don't use the dialog, but want to be able to maintain the semantics of a mandatory update.